src/test/java/org/gridsuite/balances/adjustment/server/BalancesAdjustmentTest.java

```diff
@@ -344,14 +339,11 @@ public void testNetworkComputationAreasCreationNoIterativeMode() {
             assertEquals(0, testNetwork.getGenerator("NNL1AA1 _generator").getTargetP(), 0.1);
             assertEquals(0, testNetwork.getGenerator("NNL2AA1 _generator").getTargetP(), 0.1);
             assertEquals(0, testNetwork.getGenerator("NNL3AA1 _generator").getTargetP(), 0.1);
-
-        } catch (IOException e) {
-            e.printStackTrace();
```

Comment:
Why ?

Reply:
Because
- if for any reason we can't read the file `classpath:workingTargetNetPositions.json` we can't do the test at all
- if there is an error, we don't want to silently just print the stacktrace and let the test pass.